### DISCUSSION TITLE
Kibana dashboard is missing

### DISCUSSION THREAD

**Discussion by shainmilan** [telekom-security/tpotce#1379] [Q&A] [answered]
Hi 

this may be a dumb question, but i cannot see the Kibana dashboard on the Tpot web. I'm trying to use the Tpot honey pot for a university project and not sure if i'm missing some important step . i would appreciate  if  someone  help me to understand why its missing . 

all i can see is 

![Screenshot from 2023-07-30 19-16-10](https://github.com/telekom-security/tpotce/assets/59168849/78875d18-91bb-4268-a57e-cf3d2a693ab5)

**Comment by kalogirev2** [comment, marked as answer]
Hello

log in as root on your machine and run the "dps.sh" command to see which service is running.

Then you can restart it with the command "systemctl restart tpot" to see if it restarts.
![image](https://github.com/telekom-security/tpotce/assets/93402399/c655d8c2-0e7c-471e-a691-ba5c923fc384)

**Reply by shainmilan** [reply]
Thank you so much for your help .  dps.sh showed me that for some reason kibana and logstash both were down . i restarted and it worked . much appreciated  :+1: 
![Screenshot from 2023-07-30 22-57-59](https://github.com/telekom-security/tpotce/assets/59168849/e463caae-a729-4666-9b13-32ae31ee02e3)

**Comment by kalogirev2** [comment]
No worries :)

With pleasure

**Comment by DevmanusHere** [comment]
![image](https://github.com/telekom-security/tpotce/assets/116114483/29a11736-ba95-4080-ad72-53f0cc32f6c5)
After doing systemctl restart tpot 
still cannot up kibana and others

**Reply by krepev3** [reply]
You may want to allocated more RAM for your instance for all application to consume as the instance should run as documented by TPOT to use 8GB RAM, same goes to other server that should have, more RAM is better performance.

**Comment by PLACIDEM** [comment]
Hello Everyone,

Is this  issue  solved yet? I have the same issue  where  four  services (ddospot, kibana, logstash, ma-data )are not  starting after  systemctl restart tpot command .

**Comment by shainmilan** [comment]
hi all

Just check your memory allocation on the TPot instance. I had the same
issue and I changed my
allocated RAM to 8GB from 4GB default ( VMware workstation ). For me, it
was solved after that.

Regards
Shain
